File: 387.cpp
class Solution {
public:
    int firstUniqChar(string s){
	int resp;
	int c[26]; 
	memset(c, 0, sizeof(c)); 

	for (int i = 0; i < s.size(); i++) {
		c[s[i] - 'a'] ++; 
	} 	
	for (int i = 0; i < s.size(); i++){
		if (c[s[i] - 'a' ] == 1) return i; 
	} 
	return -1; 
	
} 


};

File: 387.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        times = [0 for _ in range(26)]
        pos = 0
        
        for letter in s: 
            times[ord(letter)-ord('a')] += 1
        
        for i, letter in enumerate(s):
            if times[ord(letter)-ord('a')] == 1:
                return i
                
        return -1
            
